### DISCUSSION TITLE
콘솔창에서 html null 에러

### DISCUSSION THREAD

**Discussion by L-cloud** [kimytsc/covid-rest-vaccine-macro#27] [Q&A] [answered]
테스트를 위해 실행을 해보려 했는데

Uncaught TypeError: Cannot read property 'html' of null

이런 에러가 발생합니다.

이유가 무엇일까요?

올바르지 않은 시간에 실행을해서 그런가요?

아니면 url이 잘못되어서 그런것 인가요?

 $('.h_title').html(`<span class="accent"><span id="organizations">0</span>개 리스트에서 잔여백신</span> 예약시도중`);

**Comment by kimytsc** [comment, marked as answer]
네이버 잔여백신 `예약 신청 페이지`(README 참고)에서 하셔야됩니다.

**Reply by L-cloud** [reply]
아 제가 잘못 하고 있었네요 감사합니다.
